src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,70 @@
+import { useRef, useEffect } from 'react';
+import { Errors } from '../../types/Errors';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  onAddTodo: (title: string) => void,
+  setError: (error: Errors) => void,
+  isLoading: boolean,
+  todoTitle: string,
```

Comment:
it is best for this component to store this state within itself like:
```
export const TodoHeader: React.FC<Props> = (props) => {
  const [todoTitle, setTodoTitle] = useState('');
```

Reply:
i am using todoTitle for error handling